src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
+
+<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page, taken from the coronavirus shielding support service. The page includes a heading, a few simple paragraphs and bullet points to explain who can use the service, and a start button labelled 'Register or update your details'.">
+
+If the start point is linked to a wider process that needs more explanation, create an ‘Apply’ section within a multipart guide. 
 
-Start pages include 4 main elements:
+For example, before a user applies for probate (the right to manage someone’s estate after they die) they need to understand whether probate is required, whether they’re the right person to apply, how the details of the will and the tax situation affect what they’re supposed to do, how to get the relevant information together and how the different outcomes affect what they'll need to do next.
 
-1. The service name: this should help people understand what your service does and whether they need to use it - [learn more about naming your service](https://www.gov.uk/service-manual/design/naming-your-service).
+It’s important that users have this context, so it makes sense to embed the start point within the multipart guide — rather than creating the start point as a separate start page, allowing users to miss the context and potentially do the wrong thing.
 
-2. A short introduction to list things that most users will need to know, for example, what your service will let them do or how much it’ll cost.
+<img src="multipart-guide-start-point.png" alt="A screenshot showing an example of a start point within a multipart guide. The page is shown as page 4 of 5 of a guide and its body includes a heading, a few simple paragraphs and bullet points to explain what the user will need to start using the service, and a start button labelled 'Apply now'.">
 
-3. The ‘Start now’ call-to-action button.
+The Carer’s Allowance team found that for their servicet, [providing a start point within a multipart guide helped get users to the right information without overwhelming them](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
 
-4. A list of other ways to access your service.
+## Creating a start point using Whitehall Publisher
 
-This is a set pattern, so you will not be able to customise it.
+For non-mainstream services, publishers in departments with access to the ‘Whitehall’ app can create a simple start page using the ‘detailed guide’ format.
 
-Read guidance on [what information should go on a start page](https://www.gov.uk/service-manual/design/govuk-content-transactions) in the Service Manual.
+Use this Markdown to create a button within a detailed guide:
 
-{{ example({group: "patterns", item: "start-pages", example: "default", html: true, nunjucks: true, open: false, titleSuffix: "second"}) }}
+```
+{button start}[Button text goes here](https://servicename.service.gov.uk/first-page-within-service){/button}
+```
```

Comment:
The markup colour-coding on this is a bit odd. Will need to see if there's a way to disable or tag this block as Markdown language?

Reply:
```suggestion
`{button start}[Button text goes here](https://servicename.service.gov.uk/first-page-within-service){/button}`
```